package.json

```diff
@@ -35,20 +36,40 @@
   },
   "homepage": "https://github.com/nasa/cumulus-message-adapter-js#readme",
   "devDependencies": {
+    "@ava/typescript": "^1.1.1",
+    "@cumulus/types": "1.24.0-alpha.0",
+    "@types/aws-lambda": "^8.10.58",
+    "@types/lodash": "^4.14.150",
+    "@types/lodash.iserror": "^3.1.6",
+    "@types/lodash.get": "^4.4.6",
+    "@types/mime-types": "^2.1.0",
+    "@types/node": "^12.12.6",
+    "@types/pump": "^1.1.0",
+    "@types/sinon": "^9.0.4",
+    "@typescript-eslint/eslint-plugin": "^2.29.0",
+    "@typescript-eslint/parser": "^2.29.0",
     "ava": "^3.8.1",
     "babel-eslint": "^10.0.3",
-    "eslint": "^6.6.0",
+    "eslint": "^7.5.0",
+    "eslint-config-airbnb-base": "^14.0.0",
+    "eslint-config-airbnb-typescript": "^7.2.1",
+    "eslint-plugin-eslint-comments": "^3.0.1",
+    "eslint-plugin-import": "^2.20.2",
+    "eslint-plugin-jsdoc": "^22.1.0",
+    "eslint-plugin-lodash": "^2.7.0",
+    "eslint-plugin-node": "^7.0.1",
+    "eslint-plugin-promise": "^4.2.1",
+    "eslint-plugin-unicorn": "^19.0.1",
     "eslint-config-airbnb": "^18.0.1",
-    "eslint-plugin-import": "^2.18.2",
-    "eslint-plugin-jsdoc": "^16.1.1",
     "eslint-plugin-jsx-a11y": "^6.2.3",
-    "eslint-plugin-node": "^10.0.0",
     "eslint-plugin-react": "^7.16.0",
     "extract-zip": "^1.6.6",
     "fs-extra": "^8.1.0",
     "lodash.clonedeep": "^4.5.0",
+    "proxyquire": "^2.1.3",
     "request": "^2.83.0",
-    "rewire": "^4.0.1"
+    "rewire": "^4.0.1",
```

Comment:
It doesn't look like rewire is still used.

Reply:
ba71706